### PR TITLE
feat: [PL-23106]: Added logoutUrl in Saml Settings

### PR DESCRIPTION
**Pull request by meenakshi-harness**
##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

Screenshot 2022-03-10 at 7 29 13 PM

### CODE REVIEW

**Comment by meenakshi-harness** [comment]
dockerize

**Comment by meenakshi-harness** [comment]
trigger cypress

**Comment by meenakshi-harness** [comment]
retrigger titlecheck

**Comment by meenakshi-harness** [comment]
retrigger titlecheck

File: src/modules/33-auth-settings/modals/SAMLProvider/views/SAMLProviderForm.tsx [hunk 1]
```diff
@@ -253,6 +254,7 @@ const SAMLProviderForm: React.FC<Props> = ({ onSubmit, onCancel, samlProvider })
                         placeholder={getString('authSettings.chooseFile')}
                         multiple
                       />
+                      <FormInput.Text name="logoutUrl" label={getString('authSettings.logoutUrl')} />
```
Comment (abhinavrastogi-harness) [inline src/modules/33-auth-settings/modals/SAMLProvider/views/SAMLProviderForm.tsx:257]: no validation added on the url?

**Review by abhinavrastogi-harness — COMMENTED** [review]
Are we not planning to show the logout url on the auth settings page itself? do we need to "edit" the saml settings to see it?

File: src/modules/33-auth-settings/modals/SAMLProvider/views/SAMLProviderForm.tsx (hunk 1 above)
Comment (meenakshi-harness) [reply src/modules/33-auth-settings/modals/SAMLProvider/views/SAMLProviderForm.tsx:257]: On BE we don't have validation yet. I am not sure what are the existing logout Urls customer might have already configured. Hence skipped the validation for know. We can add it incrementally.

**Comment by meenakshi-harness** [comment]
> Are we not planning to show the logout url on the auth settings page itself? do we need to "edit" the SAML settings to see it?

From the current requirement yes we will have this setting in SAMl Setting itself. Kapil Digani can share more details if we want to show logout Url as authentication setting or not. Will check with him

**Review by abhinavrastogi-harness — APPROVED** [review]

**Review by reetika04 — APPROVED** [review]

**Pull request merged**